build.sbt

```
@@ -8,7 +8,7 @@ ThisBuild / version := "1.0-SNAPSHOT"
 ThisBuild / scalacOptions := Seq(
   "-encoding",
   "UTF-8",
-  "-target:jvm-1.8",
+  "-target:jvm-11",
```

Small nit, but `-target` got deprecated with https://github.com/scala/scala/pull/9982, now `-release` is preferred!

```suggestion
  "-release:11",
```

See https://github.com/guardian/frontend/commit/8405bfc2775469119e31f2b0264ab8ccbc9af0a5 for another example of this.

---

Thanks @rtyley, TIL!

---

Ah, apologies, I see that with 090b9c899b367d08ce1b71a6ea0688a62c5fb578 you've addressed the fact that the `-release` flag wasn't added with https://github.com/scala/scala/pull/9982 until Scala v2.13.9, and this repo was on Scala 2.13.7! Sorry, didn't mean to delay you...

---

No worries – could have chosen to defer, but felt it was worth bumping anyhow!